src/globalStyles.js

```diff
@@ -0,0 +1,11 @@
+import { createGlobalStyle } from 'styled-components';
+
+const GlobalStyle = createGlobalStyle`
+  body {
+    margin: 0;
+    padding: 0;
+    background: teal;
+    font-family: Open-Sans, Helvetica, Sans-Serif;
```

Comment:
This should be Montserrat I guess

Reply:
Oh so sorry i install the fonts via links in my index.html